app/views/users/show.html.erb

```diff
@@ -13,7 +13,7 @@
 
     <div class='button_group'>
       <%= link_to 'See all posts', user_posts_path(user_id: @user.id), class: 'btn' %>
-      <%= link_to 'Click to go back', users_path, class: 'back_btn' %>
+      <%= link_to 'Back to main', users_path, class: 'back_btn' %>
     </div>
   <% else %>
     <p>No posts found.</p>
```

Comment:
- [ ] [OPTIONAL] I think a button to add a new post should be available here to enable users to create posts for the first time. I didn't see the button to create a post until I created one, that's when a button was visible.